### ISSUE TITLE
Datetime serialization wrong for datetimes with microseconds smaller than 100000

### ISSUE THREAD

**Issue by Lukas0907** [ijl/orjson#31]
Hi,

I think there is an error in the way microseconds are serialized:

```
>>> orjson.dumps(datetime.datetime.now().replace(microsecond=10))
b'"2019-09-18T15:26:25.10"'
>>> datetime.datetime.now().replace(microsecond=10).isoformat()
'2019-09-18T15:28:28.000010'
```

Microseconds are usually given as fractional seconds. Therefore ".10" has a different meaning than ".000010".

Lukas.

**Comment by ijl** [comment]
Thank you for reporting that. It's such an odd oversight. There is now a test that the output parses to a datetime object with the same microseconds. The issue has been fixed in 2.0.8.

**Comment by Lukas0907** [comment]
Thank you for the fast response and release! Very much appreciated.
